### PR TITLE
Remove namespace attribute from ClusterRoleBinding fix #5795

### LINKED ISSUE #5795: ClusterRoleBinding tekton-pipelines-resolvers has namespace set
ClusterRoleBindings are not namesapce scoped, but the [`tekton-pipelines-resolvers` ClusterRoleBinding has a namespace set](https://github.com/tektoncd/pipeline/blob/c6ba630e38ed9766ee148b11c18fdc01a20dab5e/config/resolvers/201-clusterrolebinding.yaml#L19).

- Tekton Pipeline version:

This "namespaced" ClusterRoleBinding made it into the release.yaml of the v0.42.0 release.

### PR DESCRIPTION
**Pull request by pst**
# Changes

I am under the assumption, that this is the source file that gets rendered into the release.yaml.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [x] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by JeromeJu** [comment]
Thanks for the PR @pst . I've been wondering why this shall not be in the reslover namespace and `pipeline/config/201-rolebinding.yaml` shall be the correct file that gets rendered to release.yaml?

File: config/resolvers/201-clusterrolebinding.yaml [hunk 1]
```diff
@@ -16,7 +16,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: tekton-pipelines-resolvers
-  namespace: tekton-pipelines-resolvers
```
Comment (JeromeJu) [inline config/resolvers/201-clusterrolebinding.yaml:19]: Thanks for the PR @pst . I've been wondering why this shall not be in the reslover namespace? And I think`pipeline/config/201-rolebinding.yaml` shall be the correct file that gets rendered to release.yaml?

Comment (Yongxuanzhang) [reply config/resolvers/201-clusterrolebinding.yaml:19]: ClusterroleBinding is not namespace scoped, as mentioned in https://github.com/tektoncd/pipeline/issues/5795

**Comment by Yongxuanzhang** [comment]
I think it may be better to rewite the PR description and commit msg, make sure you follow the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)

**Review by dibyom — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**